build.gradle

```diff
@@ -12,7 +12,29 @@ subprojects {
     apply plugin: 'kotlin'
 
     dependencies {
+        def coreVersion = '246cef90b31d6fef3ecf00aaeb9ad54ae8c51850'
```

Comment:
moved from client/build.gradle
gradle could actually handle the core tests as a transitive dependency but the idea gradle plugin failed horribly...